### PR TITLE
Release/1.6.0

### PR DESCRIPTION
**Pull request by cqc-melf**
update pytket to 1.6 and pytket-quanitnuum 0.8.0

### CODE REVIEW

**Comment by cqc-melf** [comment]
> regression

Thank you, if you can confirm that this is not an issue in the notebook I an take over the investigations of the issue in tket, too.

**Comment by cqc-alec** [comment]
This is a bug in TKET: https://github.com/CQCL/tket/issues/514 .

File: examples/python/comparing_simulators.py [hunk 1]
```diff
@@ -137,13 +137,13 @@
 model = NoiseModel()
 dep_err = depolarizing_error(0.04, 2)
 for i, j in combinations(range(3), r=2):
-    model.add_quantum_error(dep_err, ["cx"], [i, j])
-    model.add_quantum_error(dep_err, ["cx"], [j, i])
+    if i != j:
```
Comment (cqc-alec) [inline examples/python/comparing_simulators.py:140]: This is unnecessary: `combinations()` returns pairs of distinct values.

**Review by cqc-alec — CHANGES_REQUESTED** [review]
Maybe we should wait for a hotfix to pytket rather than lower the optimization level on the passes here.

**Comment by cqc-melf** [comment]
I will close this and make a new PR with pytket 11.6.1

**Pull request closed without merging**